lib/index-basic.js

```diff
@@ -15,4 +15,7 @@ Plotly.register([
     require('./pie')
 ]);
 
+// locales
+Plotly.register(require('./locales.js'));
```

Comment:
Maybe it would better to put these `Plotly.register` calls in [`src/core.js`](https://github.com/plotly/plotly.js/blob/master/src/core.js) so that we don't have to duplicate them?

That would also mean moving locale modules to a new `src/locales/` directory, as we wouldn't want to require `lib/` files from `src/` right?